docker/compose.devcontainer.yaml

```
@@ -161,11 +161,32 @@
         target: /var/lib/postgresql/data
         type: volume
+  redis:
+    ports:
+    - host_ip: ${REDIS_MAPPED_HOST_IP:?error}
+      name: redis
+      published: ${REDIS_MAPPED_PORT:?error}
+      target: 6379
 # https://docs.docker.com/reference/compose-file/volumes/
+  redis-test:
+    ports:
+    - host_ip: ${REDIS_MAPPED_HOST_IP:?error}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)